### PR TITLE
[MAINTENANCE]  Rename `GE` to `GX` across codebase (GREAT-1352)

### PR DESCRIPTION
**Pull request by cdkini**
Changes proposed in this pull request:
- Rename `GE` to `GX` (where applicable)

### Definition of Done
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code

### CODE REVIEW

**Comment by ghost** [comment]
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=6494&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=6494&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=6494&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map Legend

File: docker/Dockerfile [hunk 1]
```diff
@@ -2,19 +2,19 @@ ARG PYTHON_DOCKER_TAG
...
-ARG GE_EXTRA_DEPS="spark,sqlalchemy,redshift,s3,gcp,snowflake"
+ARG GX_EXTRA_DEPS="spark,sqlalchemy,redshift,s3,gcp,snowflake"
 
 ENV PYTHONIOENCODING utf-8
 ENV LANG C.UTF-8
 ENV HOME /root
 ENV PATH /usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:${HOME}/.local/bin
-# Path where the root of the GE project will be expected
+# Path where the root of the GX project will be expected
 ENV GE_HOME /usr/app/great_expectations
```
Comment (donaldheppner) [inline docker/Dockerfile]: missed one?

File: docker/Dockerfile [hunk 2]
```diff
@@ -29,7 +29,7 @@ COPY . /tmp/great_expectations_install
 
 RUN mkdir -p /usr/app ${HOME} && \
     cd /tmp/great_expectations_install && \
-    pip install .[${GE_EXTRA_DEPS}] && \
+    pip install .[${GX_EXTRA_DEPS}] && \
     rm -rf /tmp/great_expectations_install
 
 WORKDIR ${GE_HOME}
```
Comment (donaldheppner) [inline docker/Dockerfile]: ditto

File: examples/integrations/airflow/README.md [hunk 3]
```diff
@@ -1 +1 @@
-This example shows how Great Expectations can protect an Airflow data pipeline from bad data and code bugs: [GE Airflow Example](https://github.com/superconductive/ge_tutorials)
+This example shows how Great Expectations can protect an Airflow data pipeline from bad data and code bugs: [GX Airflow Example](https://github.com/superconductive/ge_tutorials)
```
Comment (donaldheppner) [inline examples/integrations/airflow/README.md]: This URL should be updated to https://github.com/great-expectations/gx_tutorials

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py [hunk 4]
```diff
@@ -1130,7 +1130,7 @@ def _prescriptive_summary(
         **kwargs,
     ):
         """
-        Rendering function that is utilized by GE Cloud Front-end
+        Rendering function that is utilized by GX Cloud Front-end
```
Comment (donaldheppner) [inline great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]: This is the new standard rendering method for expectations (not just for Cloud), this comment should be updated. @NathanFarmer for thoughts

File: great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py [hunk 5]
```diff
@@ -383,7 +383,7 @@ def _prescriptive_summary(
         **kwargs,
     ):
         """
-        Rendering function that is utilized by GE Cloud Front-end
+        Rendering function that is utilized by GX Cloud Front-end
```
Comment (donaldheppner) [inline great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: ditto

File: great_expectations/util.py [hunk 6]
```diff
@@ -1756,7 +1756,7 @@ def get_context(
 
     if ge_cloud_mode and not config_available:
         raise GXCloudConfigurationError(
-            "GE Cloud Mode enabled, but missing env vars: GE_CLOUD_ORGANIZATION_ID, GE_CLOUD_ACCESS_TOKEN"
+            "GX Cloud Mode enabled, but missing env vars: GE_CLOUD_ORGANIZATION_ID, GE_CLOUD_ACCESS_TOKEN"
```
Comment (donaldheppner) [inline great_expectations/util.py:1759]: I think we should include support for both `GE_` and `GX_` env vars. What do you think about renaming the parameters? support both for a time and then retire the `ge_` versions?

This can be a separate PR

File: tests/data_context/test_data_context_data_docs_api.py [hunk 7]
```diff
@@ -239,7 +239,7 @@ def test_existing_local_data_docs_urls_returns_single_url_from_customized_local_
 ):
     empty_directory = str(tmp_path_factory.mktemp("yo_yo"))
     DataContext.create(empty_directory)
-    ge_dir = os.path.join(empty_directory, DataContext.GE_DIR)
+    ge_dir = os.path.join(empty_directory, DataContext.GX_DIR)
```
Comment (donaldheppner) [inline tests/data_context/test_data_context_data_docs_api.py:242]: variable name?

**Review by donaldheppner — CHANGES_REQUESTED** [review]

File: docker/Dockerfile (hunk 1 above)
Comment (cdkini) [reply docker/Dockerfile]: updated!

File: docker/Dockerfile (hunk 2 above)
Comment (cdkini) [reply docker/Dockerfile]: updated!

File: examples/integrations/airflow/README.md (hunk 3 above)
Comment (cdkini) [reply examples/integrations/airflow/README.md]: updated here and across the codebase

File: great_expectations/util.py (hunk 6 above)
Comment (cdkini) [reply great_expectations/util.py:1759]: yup this is slated for the next PR (the last in this series). cc @dctalbot

Comment (cdkini) [reply great_expectations/util.py:1759]: going to keep the change here as we'll update the other items shortly

File: tests/data_context/test_data_context_data_docs_api.py (hunk 7 above)
Comment (cdkini) [reply tests/data_context/test_data_context_data_docs_api.py:242]: fixed! note that we'll have to do this in other spots (which might be a more focused effort)

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py (hunk 4 above)
Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]: For now I would just remove this docstring entirely. We haven't added any docstrings to these private renderer methods except for this short blurb that I know of.

File: great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py (hunk 5 above)
Comment (NathanFarmer) [reply great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: For now I would just remove this docstring entirely. We haven't added any docstrings to these private renderer methods except for this short blurb that I know of.

Comment (cdkini) [reply great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]: removed 👍🏽

**Review by donaldheppner — APPROVED** [review]
